workspaces/ballerina/ballerina-visualizer/src/views/BI/Forms/FormGeneratorNew/index.tsx

```diff
@@ -205,6 +205,41 @@ export function FormGeneratorNew(props: FormProps) {
         });
     };
 
+    const defaultType = (): Type => {
+        if (typeEditorState.field?.type === 'PARAM_MANAGER') {
+            return {
+                name: typeEditorState.newTypeValue || "MyType1",
```

Comment:
can we use MyType as the default value here, and in Class as well

Reply:
update with https://github.com/wso2/vscode-extensions/pull/371/commits/e975413de0c51834c30716e56d87d0e88eef6f3c